src/pages/ProductDetail/components/Detail/DetailHeader.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './DetailHeader.scss';
+
+const DetailHeader = () => {
+  return (
+    <div className="detailHeader">
+      <h3 className="good"> CLOSET / FIG</h3>
+      <p>
+        제주 차밭의 어린 찻잎으로 만든 작설차를 원하는 용량만큼 우려드실 수 있는
+        잎차입니다. 합리적인 가격에 즐겨보세요.
+      </p>
```

Comment:
```suggestion
      <h3 className="goods"> CLOSET / FIG</h3>
      <p>
        제주 차밭의 어린 찻잎으로 만든 작설차를 원하는 용량만큼 우려드실 수 있는
        잎차입니다. 합리적인 가격에 즐겨보세요.
      </p>
```
상품이라면 goods가 될것같습니다.
그리고 타이틀하고 제품 설명하고 싱크가 안맞네요?

Reply:
제품 설명은 백엔드에서 데이터를 받아온다고 생각해서 일단 오설록에 있는 제품 설명을 가져왔는데 바꾸는게 좋을까요?
className은 수정했습니다.